packages/dev-approuter/package.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "dev-approuter",
+    "version": "0.0.1",
```

Comment:
Use the version `0.0.0` as a start version

Reply:
Ok